autotest/ogr/ogr_refcount.py

```diff
@@ -86,12 +86,10 @@ def test_ogr_refcount_2():
 def test_ogr_refcount_3():
```

Comment:
Marking the test as follows should squelch the warning while retaining coverage on `Release`:

```
@pytest.mark.filterwarnings("ignore::DeprecationWarning")
```

Reply:
good suggestion, done